### PR TITLE
fix: exclude libraries with unsupported methods

### PR DESCRIPTION
**Pull request by diegomarquezp**
Some libraries have stream-based endpoints for which some retry settings will expect non-existing methods. This adds a list of these libraries so they are not generated

### CODE REVIEW

File: spring-cloud-generator/excluded_libs.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# this is a list of libraries to be ignored by generate-library-list.sh
+google-cloud-speech
+google-cloud-dialogflow
+google-cloud-dataproc
+google-cloud-dataplex
```
Comment (zhumin8) [inline spring-cloud-generator/excluded_libs.txt:5]: Does `apigee-connect` mentioned in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1407#issuecomment-1363280649 belong to this list?

File: spring-cloud-generator/generate-library-list.sh [hunk 2]
```diff
@@ -64,6 +64,11 @@ for d in ./google-cloud-java/*java-*/; do
     echo "$artifact_id is already present in manual modules."
     continue
   fi
+  # checks if library is in the exclusion list
+  if [[ $(cat excluded_libs.txt | tail -n+2 | grep $artifact_id | wc -l) -ne 0 ]] ; then
+    echo "$artifact_id is in the exclusion list"
+    continue
+  fi
```
Comment (zhumin8) [inline spring-cloud-generator/generate-library-list.sh:71]: Can you also populate the [library_list.txt](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/b0846c64d254044fc6f3deab7fff3de6a77f2d73/spring-cloud-generator/library_list.txt) with these removals?
This script is not run within `generate-all.sh` with current setup.

File: spring-cloud-generator/excluded_libs.txt (hunk 1 above)
Comment (diegomarquezp) [reply spring-cloud-generator/excluded_libs.txt:5]: Will check

File: spring-cloud-generator/generate-library-list.sh (hunk 2 above)
Comment (diegomarquezp) [reply spring-cloud-generator/generate-library-list.sh:71]: Will do

Comment (diegomarquezp) [reply spring-cloud-generator/generate-library-list.sh:71]: Done

**Comment by emmileaf** [comment]
@diegomarquezp Thanks for catching this and the libraries that are affected! Dropping in a note from our sync - working on a minimum viable patch for this issue in https://github.com/googleapis/gapic-generator-java/pull/1212 that Diego helped verify in the workflow. We should be able unblock these libraries for preview when that fix is merged.

File: spring-cloud-generator/excluded_libs.txt (hunk 1 above)
Comment (diegomarquezp) [reply spring-cloud-generator/excluded_libs.txt:5]: It should, but this PR will likely get closed - see https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1423#issuecomment-1371508894

**Comment by diegomarquezp** [comment]
Will keep open until https://github.com/googleapis/gapic-generator-java/pull/1212 is merged

**Comment by emmileaf** [comment]
Closing as https://github.com/googleapis/gapic-generator-java/pull/1212 has been merged - thanks!

**Pull request closed without merging**